### PR TITLE
Content layout

### PR DESCRIPTION
**Pull request by martinremis**
## Summary
Add some other cards and a new title.

## Screenshot
content-layout

## Trello Card
https://trello.com/c/FuC2uY2A/23-content-layout

### CODE REVIEW

File: scss/tech.scss [hunk 1]
```diff
@@ -1,48 +1,52 @@
 .technologies {
```
Comment (FrankIglesias) [inline scss/tech.scss:1]: Check indentation

File: scss/titles.scss [hunk 2]
```diff
@@ -2,5 +2,17 @@
   display: flex;
   flex-flow: row nowrap;
   justify-content: flex-start;
-  margin: 2em;
+  margin: 1em 2em;
```
Comment (FrankIglesias) [inline scss/titles.scss]: use px

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by martinremis** [comment]
Fixed some issues from previous PRs: "alt" tag for images and new usage of a title.

File: index.html [hunk 3]
```diff
@@ -15,10 +15,63 @@ <h1 class="title">Front-End Tools</h1>
...
+        <p class="text-small">Elm is a domain-specific functional programming language for declaratively creating web browser-based graphical user interfaces.</p>
+      </div>
+      <div class="card">
+        <img src="assets/polymer_icon.svg" class="tech-logo" alt="polymer icon"/>
+        <h3 class="text-medium">Polymer</h3>
+        <h4 class="text-small bold">Release: 2015</h4>
+        <p class="text-small">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+      </div>
+    </div>
+    <div>
```
Comment (FrankIglesias) [inline index.html]: I think you don't need this div wrapper

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: scss/styles.scss [hunk 4]
```diff
@@ -2,14 +2,6 @@
...
-  align-self: center;
-  font-size: 1.5em;
-  margin-left: -33px;
-}
-
-.titles-img {
-  height: 90px;
-  width: 100px;
-  flex: 0 1;
+.bold {
```
Comment (FrankIglesias) [inline scss/styles.scss]: move this to text file

File: scss/tech.scss [hunk 5]
```diff
@@ -1,48 +1,52 @@
...
-  align-items: center;
-  width: 200px;
-  height: 284px;
-  text-align: center;
-  border-radius: 5px;
-  margin: 2em;
-}
+	display: flex;
+	flex-flow: row wrap;
+	margin: 0 7em 4em 7em;
```
Comment (FrankIglesias) [inline scss/tech.scss]: use px instead of em

File: scss/tech.scss [hunk 6]
```diff
@@ -1,48 +1,52 @@
...
+	font-size: 24px;
+	margin: .5em;
+ }
 
 .text-medium {
-  font-size: 16px;
-  margin: .5em;
-}
+	font-size: 16px;
+	margin: .5em;
```
Comment (FrankIglesias) [inline scss/tech.scss]: same

File: scss/tech.scss [hunk 7]
```diff
@@ -1,48 +1,52 @@
...
+	margin: .2em;
+	max-width: 100px;
+ }
 
 .text-big {
-  font-size: 24px;
-  margin: .5em;
-}
+	font-size: 24px;
+	margin: .5em;
```
Comment (FrankIglesias) [inline scss/tech.scss]: same

File: scss/tech.scss [hunk 8]
```diff
@@ -1,48 +1,52 @@
...
+ }
 
 .text-small {
-  font-size: 12px;
-  margin: .5em;
-}
+	font-size: 12px;
+	margin: .5em;
+ }
```
Comment (FrankIglesias) [inline scss/tech.scss]: Check indentation

File: scss/titles.scss [hunk 9]
```diff
@@ -2,5 +2,17 @@
...
   flex-flow: row nowrap;
   justify-content: flex-start;
-  margin: 2em;
+  margin: 16px 32px;
+}
+
+.title {
+  align-self: center;
+  font-size: 24px;
+  margin-left: -33px;
```
Comment (FrankIglesias) [inline scss/titles.scss]: Why this margin?

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by FrankIglesias** [comment]
Also resolve conflicts

**Review by FrankIglesias — APPROVED** [review]
Excelent job!

**Pull request merged**